user-images/perftest/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+ARG D_RDMA_CORE_TAG=v31.0
+ARG D_PERFTEST_TAG=4.4-0.29
+
+WORKDIR /root
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install net-tools iproute2 apt-utils git autotools-dev autoconf libtool pciutils
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install build-essential cmake gcc libudev-dev libnl-3-dev \
+    libnl-route-3-dev ninja-build pkg-config valgrind python3-dev cython3 python3-docutils pandoc
+
+# Compile and install rdma-core
+RUN git clone --branch ${D_RDMA_CORE_TAG} https://github.com/linux-rdma/rdma-core.git
+RUN /bin/bash -c 'cd /root/rdma-core && ./build.sh && cd build && cp -R lib/* /usr/lib && cp -R etc/* /etc/ && cp -R include/* /usr/include/'
+# Compile and install perftest
+RUN git clone --branch ${D_PERFTEST_TAG} https://github.com/linux-rdma/perftest.git
```

Comment:
it better to do 2 stage docker. 1- build and there other one is just the binary

Reply:
sure will do a dual stage.